### PR TITLE
Improved relationship components for CLEM

### LINKED ISSUE #5063: Remove variable tracking from relationship and add new tracker component
The CLEM relationship component contains all the variables and code for tracking a variable based on the relationship. This is unneeded and causes validation issues for simple relationships. It is proposed to simplify the relationship component and add a new relationship tracker component.

### PR DESCRIPTION
**Pull request by lie112**
Resolves #5063

### CODE REVIEW

File: Models/CLEM/Activities/LabourAERelationship.cs [hunk 1]
```diff
@@ -12,79 +12,79 @@
...
-    [Version(1, 0, 1, "")]
-    [HelpUri(@"content/features/Resources/Labour/LabourAdultEquivalentRelationship.htm")]
-    public class LabourAERelationship: Relationship, IValidatableObject
-    {
-        /// <summary>
-        /// Starting value
-        /// </summary>
-        [XmlIgnore]
-        public new double StartingValue { get; set; }
+    ///// <summary>
```
Comment (hol430) [inline Models/CLEM/Activities/LabourAERelationship.cs:15]: Why not just get rid of this if we don't need it anymore? Same for Relationship.cs and some of RelationshipTracker.cs.

Comment (lie112) [reply Models/CLEM/Activities/LabourAERelationship.cs]: oops. missed deleting that one.

**Comment by lie112** [comment]
retest please jenkins

**Comment by lie112** [comment]
I am unable to resolve this problem.

**Comment by lie112** [comment]
Any ideas what the problem is here? I have pulled the latest master changes. The errors re in the validation, not problems with CLEM simulations as far as I can tell.

**Comment by hol430** [comment]
Sorry, this is probably due to my recent meddling with the performance tests code. I will investigate.

**Comment by hol430** [comment]
retest this please jenkins

File: Models/CLEM/RelationshipTracker.cs [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+            if (Maximum < Minimum)
+            {
+                string[] memberNames = new string[] { "Maximum" };
+                results.Add(new ValidationResult("The maximum running value must be greater than the Minimum value", memberNames));
+            }
+            //if (XValues == null)
+            //{
+            //    string[] memberNames = new string[] { "XValues" };
+            //    results.Add(new ValidationResult("X values are required for relationship", memberNames));
+            //}
```
Comment (hol353) [inline Models/CLEM/RelationshipTracker.cs:96]: Can we remove the commented code please?

Comment (lie112) [reply Models/CLEM/RelationshipTracker.cs]: Sorry, too easy to loose track of workings.

**Pull request merged**